seshdash/views.py

```diff
@@ -96,18 +96,22 @@ def index(request,site_id=0):
     # Create an object of the get_high_chart_date
     context_dict['high_chart']= get_high_chart_data(request.user,site_id,sites)
     context_dict['site_id'] = site_id
-    print "trying to connect to influx"
     #Generate measurements in the time_series_graph
     client=Influx()
     measurements_value=client.get_measurements()
-    print "connected to influx"
     measurements =[]
+    measures = {}
 
     if measurements_value is not None:
         for measurement in measurements_value:
             measurements.append(measurement['name'])
 
-    context_dict['measurements']= measurements
+    for measure in measurements:
+        if measure == "soc" or measure == "pv_production" or measure == "AC_Load_in" or measure == "battery_voltage":
```

Comment:
This won't work. It's always best not to hardcode values like these. You should do this based on the aarray in model not harcoded.